people/tomasp.json

```diff
@@ -0,0 +1,10 @@
+{
+  "key": "tomasp",
+  "name": "Mgr. Tomáš Pavlín",
+  "nick": "Tomáš",
+  "email": "tomas.pavlin@mild.blue",
+  "title_cs": "Full-stack vývojář & produktový manažer",
+  "title_en": "Full-stack developer & product manager",
+  "short_desc_cs": "Tomáš má vystudovaný obor Umělá inteligence na MFF UK a má dlouholeté zkušenosti s vývojem aplikací využívajících metod strojového učení. Snadno si poradí s frontendem i backendem, ale je také zodpovědný za projektovou stránku našich produktů.",
+  "short_desc_en": "Tomas holds a Master's degree in artificial intelligence and he has many years of experience building applications that use machine learning approaches. While Tomas is skilled with both frontend and backend development, he is also responsible for project management of our products."
```

Comment:
myslím, žes to napsal hezky. 
Tenhle koment je tady, jen abych zjistila, jak se to dělá :-)
Magda